README.md

```diff
@@ -22,4 +22,5 @@ the output.
 ```
 $ rbenv each bundle install
 $ rbenv each -v rake test
+$ rbenv each --skip-aliases bundle check
```

Comment:
This usage example isn't correct, right? `rbenv each` doesn't accept `--skip-aliases`.